[2025-09-08T17:04:13.809+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:04:13.816+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA scheduled__2025-08-23T12:00:00+00:00 [queued]>
[2025-09-08T17:04:13.820+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA scheduled__2025-08-23T12:00:00+00:00 [queued]>
[2025-09-08T17:04:13.820+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:04:13.833+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskA> on 2025-08-23 12:00:00+00:00
[2025-09-08T17:04:13.836+0000] {standard_task_runner.py:72} INFO - Started process 17479 to run task
[2025-09-08T17:04:13.838+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskA', 'scheduled__2025-08-23T12:00:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmplm5cme3s']
[2025-09-08T17:04:13.839+0000] {standard_task_runner.py:105} INFO - Job 124: Subtask taskA
[2025-09-08T17:04:13.871+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskA scheduled__2025-08-23T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:04:13.917+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-08-23T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-23T12:00:00+00:00'
[2025-09-08T17:04:13.918+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:04:13.929+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:04:13.929+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed!']
[2025-09-08T17:04:13.935+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:04:13.936+0000] {subprocess.py:93} INFO - TASK A has executed!
[2025-09-08T17:04:13.937+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:04:13.951+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:04:13.952+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskA, run_id=scheduled__2025-08-23T12:00:00+00:00, execution_date=20250823T120000, start_date=20250908T170413, end_date=20250908T170413
[2025-09-08T17:04:13.971+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:04:13.989+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:04:13.993+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
